app/presenters/nsm/check_answers/application_status_card.rb

```diff
@@ -121,22 +128,35 @@ def update_claim
         ].compact
       end
 
+      # rubocop:disable Metrics/AbcSize
       def response
         @response ||= if claim.submitted?
                         []
                       elsif claim.granted? && claim.assessment_comment.blank?
                         [I18n.t('nsm.steps.view_claim.granted_response')]
                       elsif claim.expired?
                         expiry_response
+                      elsif claim.sent_back? && claim.pending_further_information.present?
+                        further_information_response + [update_claim_button]
                       else
                         claim.assessment_comment.split("\n")
                       end
       end
+      # rubocop:enable Metrics/AbcSize
 
       def expiry_response
-        I18n.t('nsm.steps.view_claim.expiry_explanations',
-               requested: claim.pending_further_information.requested_at.to_fs(:stamp),
-               deadline: claim.pending_further_information.resubmission_deadline.to_fs(:stamp))
+        application_helpers.sanitize_strings(I18n.t('nsm.steps.view_claim.expiry_explanations',
```

Comment:
We really shouldn't need to sanitize here or below - this isn't user-entered content. We just need to mark the strings as `.html_safe`, which saves a bunch of overhead of string processing